src/plugins/legacy/itkDataImage/interactors/medVtkViewItkDataImage4DInteractor.cpp

```diff
@@ -122,11 +122,12 @@ void medVtkViewItkDataImage4DInteractor::setInputData(medAbstractData *data)
 
         if (SetViewInput(data, layer) )
         {
+            double frameRate = static_cast<double>(m_poConv->getNumberOfVolumes()) / static_cast<double>(m_poConv->getTotalTime());
```

Comment:
Just for info, there is no need to cast both operands to double. Only one is sufficient (a division involving a double results in a double).

Reply:
True i just did a copy paste without thinking do you want me to modify it?

Reply:
no don't worry it's just something I see often in the code so I wanted to point it out to